src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -127,19 +127,29 @@ PathTuples joinPredicateAndObject(VarOrPath predicate, ObjectList objectList) {
   return tuples;
 }
 
+namespace {
+// Converts the PrefixMap to the legacy data format used by ParsedQuery
+vector<SparqlPrefix> convertPrefixMap(
+    const SparqlQleverVisitor::PrefixMap& map) {
+  vector<SparqlPrefix> prefixes;
+  for (auto const& [label, iri] : map) {
+    prefixes.emplace_back(label, iri);
+  }
+  return prefixes;
+}
+}  // namespace
+
```

Comment:
I think that in the meantime, the `prefixMap` that is stored in the `ParsedQuery` is not used anymore and can be completely removed, including all the conversion functions to and from it.:

EDIT: The only usage is in the parsing of the Filters, but the member of the ParsedQuery class should still be unused, can you please check this, and add the comment here `//TODO<joka921, qup42> remove this functionality as soon as the FILTERs are also parsed using ANTLR`

Reply:
The filter parsing already uses `SparqlQleverVisitor::PrefixMap`.  So this code is can be removed at this point.